model/vm.rb

```diff
@@ -11,6 +11,7 @@ class Vm < Sequel::Model
   one_to_one :assigned_vm_address, key: :dst_vm_id, class: :AssignedVmAddress
   one_to_many :vm_storage_volumes, key: :vm_id, order: Sequel.desc(:boot)
   one_to_one :active_billing_record, class: :BillingRecord, key: :resource_id do |ds| ds.active end
+  one_to_many :pci_device, key: :vm_id, class: :PciDevice
```

Comment:
Isn't it a `one_to_one` relation? If a vm can have multiple PciDevice, IMHO, we need to rename is as `:pci_devices`.

Reply:
A vm can have multiple PciDevices, renamed to `:pci_devices`